.travis.yml

```diff
@@ -113,7 +114,7 @@ deploy:
  - provider: releases
    file_glob: true
    file: dist/activitywatch-*.*
-   api_key:
+   token:
```

Comment:
Why change this?

Reply:
IIRC this was just because the .travis linter was complaining about the older api_key key.